cwms-data-api/src/test/resources/cwms/cda/data/dto/basin/basin.json

```diff
@@ -0,0 +1,46 @@
+{
+  "basinName": "TEST_BASIN",
+  "officeId": "TEST_OFFICE",
+  "primaryStream": {
+
+  },
+  "sortOrder": 1.0,
+  "basinArea": 100.0,
+  "contributingArea": 100.0,
+  "parentBaseId": "TEST_PARENT",
+  "streams": {
```

Comment:
update this json to reflect changes to DTO above.

Reply:
Addressed changes